pom.xml

```diff
@@ -263,7 +268,11 @@
                 <plugin>
                     <groupId>org.apache.maven.plugins</groupId>
                     <artifactId>maven-deploy-plugin</artifactId>
-                    <version>2.8.2</version>
+		    <version>2.8.2</version>
```

Comment:
Formatting looks off.

Reply:
ugh...apparently the google codestyle i added has xml which is vastly different than what we have